benchmarking/src/performance_evaluation.py

```diff
@@ -718,66 +749,13 @@ def build_prompt(
-        prompt_template = "Generate a detailed, extensive and complex movie script of the Star Wars movie 'The Empire strikes back', describing how every character felt, environment details and onomatopoeias. After that, generate a complete and extensive end-to-end plot of the movie. Finally, generate a detailed explanation of each character biograhy in multiple paragraphs. "
-
-        # Prompt for Mistral models
-        if utils.MODEL_TYPE_IDENTIFIER["mistral"] in self.model_name.lower():
-
-            prompt_content = self.generate_prompt_content(
-                prompt_template,
-                num_input_tokens,
-            )
-
-            full_input_prompt = "[INST]" + prompt_content + "[/INST]"
-
-        # Prompt for Llama3 models
-        elif utils.MODEL_TYPE_IDENTIFIER["llama3"] in self.model_name.lower():
-
-            system_prompt = f"<|begin_of_text|><|start_header_id|>system<|end_header_id|>{sys_prompt_template}<|eot_id|>"
-            prompt_content_length = num_input_tokens - self.get_token_length(
-                system_prompt
-            )
-
-            prompt_content = self.generate_prompt_content(
-                prompt_template, prompt_content_length
-            )
-
-            full_input_prompt = (
-                system_prompt
-                + "<|start_header_id|>user<|end_header_id|>"
-                + prompt_content
-                + "<|eot_id|><|start_header_id|>assistant<|end_header_id|>"
-            )
-
-        # Prompt for Deepseek models
-        elif utils.MODEL_TYPE_IDENTIFIER["deepseek"] in self.model_name.lower():
-
-            system_prompt = f"{sys_prompt_template}"
-            prompt_content_length = num_input_tokens - self.get_token_length(
-                system_prompt
-            )
-
-            prompt_content = self.generate_prompt_content(
-                prompt_template, prompt_content_length
-            )
-
-            full_input_prompt = system_prompt + prompt_content
-
-        # Prompt for Llama2 and other models
-        else:
-
-            system_prompt = f"[INST]<<SYS>>{sys_prompt_template}<</SYS>>"
-            prompt_content_length = num_input_tokens - self.get_token_length(
-                system_prompt
-            )
-
-            prompt_content = self.generate_prompt_content(
-                prompt_template, prompt_content_length
-            )
-
-            full_input_prompt = system_prompt + prompt_content + "[/INST]"
+        sys_prompt_template = f"You are a helpful assistant that generates movie scripts. "
+        prompt_template = "Generate a detailed, extensive and complex movie script of the Star Wars movie The Empire strikes back, describing how every character felt, environment details and onomatopoeias. After that, generate a complete and extensive end-to-end plot of the movie. Finally, generate a detailed explanation of each character biograhy in multiple paragraphs. "*500
```

Comment:
Could we put these prompt templates in yamls for easier maintenance @snova-rodrigom?